debug/customization/vector-tile-layer/smk-config.json

```diff
@@ -0,0 +1,124 @@
+{
+    "name": "vector_layer_demo",
+    "viewer": {
+        "type": "leaflet",
+        "location": {
+            "extent": [
+                -140.80078125000003,
+                61.62728646147466,
+                -108.72070312500001,
+                46.13417004624326
+            ],
+            "center": [
+                -124.76074218750003,
+                51.88072825385896
+            ],
+            "zoom": 6
+        },
+        "baseMap": "StamenTonerLight"
+    },
+    "layers": [],
+    "tools": [
+        {
+            "type": "toolbar",
+            "enabled": true
+        },
+        {
+            "type": "list-menu",
+            "enabled": true,
+            "title": "Vector Layer Demo!"
+        },
+        {
+            "type": "pan",
+            "enabled": true
+        },
+        {
+            "type": "zoom",
+            "enabled": true
+        },
+        {
+            "type": "about",
+            "enabled": true,
+            "title": "About Vector Layer Demo!",
```

Comment:
If we're going to have the About tool enabled, we should add a helpful description.

Reply:
Er, yes, I forgot to update this after demoing this internally ... I'll add something more appropriate :)